src/types/runtypes.ts

```diff
@@ -0,0 +1,26 @@
+import { String, Number, InstanceOf, Record } from 'runtypes';
+
+// Runtype for request body with DOB
+const RequestBodyWithDOB = Record({
+    name: String,
+    dob: InstanceOf(Date),
+    phoneNo: Number,
+    email: String,
+    password: String
+});
+
+// Runtype for request body without DOB
+const RequestBodyWithoutDOB = Record({
```

Comment:
instead of defining twice try to change the dob  type as Optional
please refer Optional in runtypes

Reply:
I was wondering if there was someway to avoid defining it twice. Thanks for the reference Sree! I'll use it.